fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -56,7 +56,7 @@ Before starting to use _match_, make sure to read the [codesigning.guide](https:
 💥  | Easily reset your existing profiles and certificates if your current account has expired or invalid profiles
 ♻️  | Automatically renew your provisioning profiles to include all your devices using the `--force` option
 👥  | Support for multiple Apple accounts and multiple teams
-✨ | Tightly integrated with [fastlane](https://fastlane.tools) to work seamlessly with [gym](https://docs.fastlane.tools/actions/gym) and other build tools
+✨ | Tightly integrated with [fastlane](https://fastlane.tools) to work seamlessly with [gym](https://docs.fastlane.tools/actions/gym/) and other build tools
```

Comment:
General question: Is this a recent change that the action docs all end with a slash?

Reply:
I think it's from the beginning. If you visit https://docs.fastlane.tools/actions/gym, a trailing slash (ugly IMHO) will be added, so I decided to add it too to links missing it. Also because they were not consistent, some of them had it, some not.